renormalizer/mps/mps.py

```diff
@@ -1190,9 +1199,21 @@ def func1(y):
                                     coef, ovlp_inv1=S_L_inv_list[imps+1],
                                     ovlp_inv0=S_L_inv_list[imps], ovlp0=S_L_list[imps])
                             return func(0, y)
+                        
```

Comment:
Is it better to wrap this routine as a function?

Reply:
Did you mean func1 is not necessary?

Reply:
I mean L1203 to L1215, which has a lot of duplication in this commit